symbolic-base/src/ZkFold/Symbolic/Class.hs

```diff
@@ -89,6 +90,10 @@ class
 embed :: (Symbolic c, Functor f) => f (BaseField c) -> c f
 embed cs = fromCircuitF hunit (\_ -> return (fromConstant <$> cs))
 
+-- | Embeds the witness value(s) into generic context @c@.
```

Comment:
I think it would be worth mentioning here that witnesses are going to be unconstrained